charts/fleet-agent/100.0.0+up0.3.6/Chart.yaml

```diff
@@ -1,8 +1,10 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.22.0-0'
```

Comment:
2.6.0  (when this chart was released) doesn't support k8s 1.22?

https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-6-0/

Should be 1.18-1.21

Released in 2.6.0 based on https://github.com/rancher/rancher/issues/32294